### ISSUE TITLE
"o build-wrap -debug true" causes JIT debugger window to appear

### ISSUE THREAD

**Issue by KonstantinNosov** [OpenWrap/openwrap#263]
from o get-help build-wrap

```
    -Debug <Boolean>
            Specifies that the package will be built using a debug configuration.
```

but when executing "o build-wrap -debug true"
the JIT debug window appears with "An unhandled exception ('Launch for user') occurred in o.exe" message

looks like o.exe code base has something like the following:

```
    if(args.contains("-debug"))
       Debugger.Launch();
```

**Comment by serialseb** [comment]
Yes that's documented behavior from the shell. May have to either change the shell to use -startdebugger or change the -debug and -release on the command to something else. I think i'd prefer the first option.

**Comment by serialseb** [comment]
Fixed in openrasta/openwrap-shell@cccbed858fe63db02cbb3296ad33e32805d4a88b
